src/database.rs

```diff
@@ -57,7 +57,10 @@ impl Database
 		let db = Self::new(path)?;
 		match db.connection.execute_batch(DATABASE_SQL)
 		{
-			Ok(_) => { return Ok(db); },
+			Ok(_) =>
+			{
+				return Ok(db);
```

Comment:
I changed this in the code due to an issue with the code coverage not counting this line when it is properly indented. Can you please revert it back to its original state ?